deploy-lighthouse-reth.py

```
@@ -0,0 +1,891 @@
+# Quickstart :: Rust Lang Clients :: Docker-free
+#
+# Made for home and solo stakers 🏠🥩
+#
+# Acknowledgments
+# Validator-Install is branched from validator-install written by Accidental-green: https://github.com/accidental-green/validator-install
+# The groundwork for this project was established through their previous efforts.
+
+import os
+import requests
+import re
+import fnmatch
```

Ruff (0.8.2)

`fnmatch` imported but unused

Remove unused import: `fnmatch`

(F401)